docs/agent/changes.md

```diff
@@ -195,13 +195,6 @@ Agent 6 in the Linux, MacOS and Windows packages.
 Similar to the Agent 5, the APM agent is enabled by default. To disable it, set
 `apm_enabled` to `false` in the main agent configuration (`datadog.yaml`).
```

Comment:
Thanks for updating the docs. Can you also update this sentence? Similar to the process agent, it should now mention the following:
```yml
apm_config:
  enabled: true
```

instead of `apm_enabled`. Note that unlike the process agent `enabled` is a boolean here :)